package.json

```diff
@@ -77,6 +77,7 @@
     "sass": "^1.49.7",
     "styled-components": "^4.4.1",
     "web3": "^1.7.1",
+    "web3-core": "^1.7.3",
```

Comment:
Since this is only used for types, it should be added to the dev dependencies.
TypeScript will complain that it needs to be added to the main dependencies when you try to import it, so you'll need to change your import to:
```
import type { TransactionReceipt } from 'web3-core';
```